README.md

```diff
@@ -50,29 +50,29 @@ Nano ID use [better algorithm] and test uniformity:
 
 ### Normal
 
-Main module uses URL-friendly symbols (`A-Za-z0-9_~`) and return ID
-with 22 characters (to have same uniqueness as UUID).
+The main module uses URL-friendly symbols (`A-Za-z0-9_~`) and returns an ID
+with 22 characters (which matches uniqueness of UUID).
 
 ```js
 var nanoid = require('nanoid')
 model.id = nanoid() //=> "Uakgb_J5m9g~0JDMbcJqLJ"
 ```
 
-Symbols `-,.()` are not encoded in URL, but in the end of link they could
-be detected as punctuation.
+Symbols such as `-,.()` aren't encoded in URL, but in the end of a link
+they could be identified as a punctuation symbol.
```

Comment:
I feel like the sentence isn't finished or lacks the conclusion why it matters that the symbols could be in interpreted as punctuation.